configman/config_manager.py

```diff
@@ -45,21 +45,50 @@
 import functools
 import warnings
 
-import configman as cm
-import converters as conv
-import config_exceptions as exc
-import value_sources
-import def_sources
-
 #==============================================================================
 # for convenience define some external symbols here - some client modules may
 # import these symbols from here rather than their origin definition location.
 # PyFlakes may erroneously flag some of these as unused
-from option import Option, Aggregation
-from dotdict import DotDict, DotDictWithAcquisition, iteritems_breadth_first
-from namespace import Namespace
-from required_config import RequiredConfig  # used elsewhere - do not remove
-from config_file_future_proxy import ConfigFileFutureProxy
+from configman import (
+    command_line
+)
```

Comment:
no problem, that one bothers me too.  once argparse is added, that one line in the __init__.py gets more complicated anyway.